quetz/tasks/workers.py

```diff
@@ -29,7 +29,7 @@ def prepare_arguments(func: Callable, **resources):
     return kwargs
 
 
-def wrapper(func, api_key, browser_session, **kwargs):
+def wrapper(func, api_key, browser_session, config, **kwargs):
```

Comment:
maybe more telling  function name and docstring would be helpful

Reply:
cool, agreed. will change.